tests/async/tasync_misc.nim

```diff
@@ -1,8 +1,3 @@
-discard """
-  exitcode: 0
-  output: "ok"
-"""
-
```

Comment:
Why remove this? I'd rather keep this in case it's relevant to some of the existing tests.

Reply:
it serves no purpose; simpler is better.